src/docs/forms/components/TextInput/examples/03-SimpleTextInput.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
```

Comment:
Split up examples for feedback icon and max length. As much as possible, please split up examples into specific property use cases instead of combining two features into a single example.